script/install_locally.sh

```diff
@@ -179,6 +179,7 @@ PYTHON_EXEC=${PYENV_VERSION_PATH}/bin/python
 POETRY_PATH=~/.poetry
 VENV_PATH=./.venv
 VENV_REPO_PATH=${VENV_PATH}/repo
+POETRY_VERSION=1.0.10
```

Comment:
Needs a comment to tell user to verify that they rm -rf ~/.poetry/ and rm -rf .venv/